File: __tests__/__fixtures__/output/javascript/fetch/jsonObj-multiline.js
const options = {
  method: 'POST',
  headers: {'content-type': 'application/json'},
  body: '{"foo":"bar"}'
};

fetch('https://httpbin.org/anything', options)
  .then(response => response.json())
  .then(response => console.log(response))
  .catch(err => console.error(err));

File: src/targets/swift/urlsession/fixtures/jsonObj-multiline.swift
import Foundation

let parameters = ["foo": "bar"] as [String : Any?]

let postData = try JSONSerialization.data(withJSONObject: parameters, options: [])

let url = URL(string: "https://httpbin.org/anything")!
var request = URLRequest(url: url)
request.httpMethod = "POST"
request.timeoutInterval = 10
request.allHTTPHeaderFields = ["content-type": "application/json"]
request.httpBody = postData

let (data, _) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))